tests/integration/tests/namespace/test-ingresses.spec.js

```diff
@@ -47,6 +47,14 @@ context('Test Ingresses', () => {
       .contains('ui5-title', NAME)
       .should('be.visible');
 
+    cy.getMidColumn()
+      .contains(/specification/i)
+      .should('be.visible');
+
+    cy.getMidColumn()
+      .contains(/tls/i)
+      .should('be.visible');
+
```

Comment:
you could add a check if the given hosts and secretName are displayed correctly :)